docs/tutorials/pdf_demo.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
We should be using hyphenated filenames in notebooks (and other files that are going to show up as doc pages). Underscores in URLs are just nasty...